spec/src/main/resources/tck-audit.xsl

```diff
@@ -0,0 +1,265 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think it is a misuse of the maven directory structure to put a XSL template into the resources folder, as it has the character of a build script, while the resources folder is intended to be packed into the result as-is. It might make sense to move the file to src/main/xsl, as it produces "source code" in the sense of book generation.

Reply:
Good point! The original intent of `src/main/resources` is for files which needs to be picked up by the `maven-resource-plugin`. I agree that moving the stylesheet to something like `src/main/xsl` is better. I'll adjust the path later today.

Reply:
I moved the file to `src/main/xsl`.